src/style.css

```diff
@@ -1,3 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  color: rgba(0, 0, 0, 1);
```

Comment:
The color `rgba(0, 0, 0, 1)` is used for text. Ensure that this color provides sufficient contrast against the background for accessibility.